libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -498,16 +523,28 @@ namespace impl {
          out(name, std::move(mvo));
       }
 
+
+      /**
+       * overload of to_variant_object for signed_block, support old signed_block_v0 format
+       */
+      template<typename Resolver>
+      static void add( mutable_variant_object &out, const char* name, const signed_block& block, Resolver resolver, abi_traverse_context& ctx )
+      {
+         // TODO: this could be faster by going directly to variant
```

Comment:
Yes, and it definitely should be. Otherwise, there is no point in adding the `add` for `packed_transaction` above.

Reply:
Finished this up.